indico/modules/events/editing/operations.py

```diff
@@ -229,3 +229,24 @@ def update_tag(tag, code=None, title=None, color=None):
 def delete_tag(tag):
     logger.info('Tag %r deleted by %r', tag, session.user)
     db.session.delete(tag)
+
+
+def update_file_type(file_type, name=None, extensions=None, allow_multiple_files=None, required=None, publishable=None):
```

Comment:
Maybe using kwargs would be better here...

```python
def update_file_type(file_type, **data):
    file_type.update_from_dict(data, keys=('name', 'extensions', ...))
```

(the `keys=...` list is not required, but I'd like to make it somewhat clear when looking at the code which kwargs are accepted)

Reply:
I have to create `update_from_dict `in the `EditingFileType` model then.

Reply:
No, it's part of `IndicoModel` which all models inherit from since `db.Model` uses it.

Reply:
Oh, I meant `populate_from_dict`, not `update_from_dict`...